src/models/Transform.js

```diff
@@ -10,20 +10,32 @@ const transformOptions = {
     min: 1
   },
   transformMethod: {
-    title: 'Transform after each',
+    title: 'When transforming shape',
     type: 'dropdown',
     choices: ['smear', 'intact'],
   },
+  growEnabled: {
```

Comment:
I don't understand why these are new. What am I missing?

Reply:
The fields are custom toggle-buttons and we didn't need an option for them to render. Now we do, because the CommentsExporter relies on options for its labels. This lets us not have to hardwire labels in both options and in getComments.

Reply:
The fields are custom toggle-buttons and we didn't need an option for them to render. Now we do, because the CommentsExporter relies on options for its labels. This lets us not have to hardwire labels in both options and in getComments.